lib/gui-utils-tft/src/TFTUtils.hpp

```diff
@@ -29,13 +29,13 @@
 #define blue 0x5D9B
 #define ligthgreen 0xF59F
 
-enum AQI_COLOR { AQI_NONE, AQI_PM, AQI_CO2 };
+enum AQI_COLOR { AQI_NONE, AQI_PM, AQI_CO2, AQI_CO, AQI_NH3 };
 
 typedef struct GUIData {
     uint8_t mainUnitId;
     uint8_t onSelectionUnit;
-    uint16_t mainValue;
-    uint16_t minorValue;
+    float mainValue;
```

Comment:
please add a photo of T-Display screen with float numbers, with these new sensors and also with a previous one, like PM2.5.

Reply:
![image](https://github.com/kike-canaries/canairio_firmware/assets/20878412/fb5d2dab-ed16-4db4-86a2-005325b38c3f)
![image](https://github.com/kike-canaries/canairio_firmware/assets/20878412/2e89f20f-6a84-450d-8350-dcb085057139)
![image](https://github.com/kike-canaries/canairio_firmware/assets/20878412/6a6b8a49-2fde-4d10-af00-b65fab09adfc)
![image](https://github.com/kike-canaries/canairio_firmware/assets/20878412/fe1f8781-e14e-4e90-adcd-709119a05068)
![image](https://github.com/kike-canaries/canairio_firmware/assets/20878412/7608466d-65ce-4055-a5dc-0a2f84c8bb59)